### ISSUE TITLE
Missing heading markup for visual heading

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3115]
The "Scene" text acts as a heading for the following section but is only emphasized  visually with no heading mark-up. Screen reader users will not be able to use heading  navigation to reach this section. 

**Suggested Solution** 
Implement "Scene" as a level 2 heading.

Severity: 2
